config/examples/delta/MKS/SBASE/Configuration.h

```diff
@@ -626,7 +626,8 @@
 
   #if EITHER(DELTA_AUTO_CALIBRATION, DELTA_CALIBRATION_MENU)
     // Set the radius for the calibration probe points - max DELTA_PRINTABLE_RADIUS for non-eccentric probes
-    #define DELTA_CALIBRATION_RADIUS 121.5 // (mm)
+    // Overrides auto-calculated value, and ignores probe offsets set using M851.
+    //#define DELTA_CALIBRATION_RADIUS DELTA_PRINTABLE_RADIUS - (MIN_PROBE_EDGE)  // (mm)
```

Comment:
Removing these defined values is throwing away established knowledge about the size of these printers.